sources/packages/web/src/views/student/StudentAssessmentAward.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <student-page-container layout-template="centered">
+    <template #header>
+      <header-navigator
+        title="Application details"
+        subTitle="View Assessment"
+        :routeLocation="{
+          name: StudentRoutesConst.STUDENT_APPLICATION_DETAILS,
+          params: {
+            id: applicationId,
+          },
+        }"
+      />
+      <detail-header :headerMap="headerData" />
+    </template>
+    <v-card class="p-4">
+      <body-header
+        title="Summary"
+        subTitle="Below is the summary from your assessment. To view your entire assessment, click on View assessment."
+      >
+        <template #actions>
+          <v-btn
+            class="float-right"
+            color="primary"
+            prepend-icon="fa:far fa-file-lines"
+            @click="goToNoticeOfAssessment"
+            >View assessment</v-btn
+          >
+        </template>
+      </body-header>
+      <assessment-award-details :assessmentAwardData="assessmentAward" />
+    </v-card>
+    <v-card class="p-4 my-3">
+      <v-row>
+        <v-col>
+          <h2 class="category-header-medium primary-color">
+            How to receive your award
+          </h2>
+          <p class="default-color my-3">
+            Please visit the
+            <span class="font-weight-bold"
+              >National Student Loan Service Centre (NSLSC)</span
+            >
+            to collect your awarded funds. It may take time to process if you do
+            not see it immediately. Please contact NSLSC if you have additional
+            questions about receiving your funds.
+          </p>
+          <v-btn
+            color="primary"
+            prepend-icon="fa:fas fa-up-right-from-square"
+            href="https://protege-secure.csnpe-nslsc.canada.ca/en/public/register-account"
+            target="_blank"
```

Comment:
Please include the attribute `rel="noopener"`. It is the Sonarcloud recommendation for security issues while redirecting to an external website.